recipes/stlab/1.7.0_and_above/conanfile.py

```diff
@@ -0,0 +1,213 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import copy, get, replace_in_file, rmdir
+from conan.tools.scm import Version
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+
+import os
+
+required_conan_version = ">=1.52.0"
+
+class Stlab(ConanFile):
+    name = 'stlab'
+    description = 'The Software Technology Lab libraries.'
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://github.com/stlab/libraries'
+    license = 'BSL-1.0'
+    topics = 'conan', 'c++', 'concurrency', 'futures', 'channels'
+
+    settings = "arch", "os", "compiler", "build_type",
+
+    options = {
+        "use_boost": [True, False],
```

Comment:
Overwhelmingly we have `with_boost`  https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md#recommended-feature-options-names